pkg/fs/apkfs.go

```diff
@@ -235,8 +235,17 @@ func (a *APKFS) Stat(path string) (fs.FileInfo, error) {
 	}
 	return &apkFSFileInfo{file: file, name: file.name[strings.LastIndex(file.name, "/"):]}, nil
 }
-
+func correctApkFSPath(path string) string {
+	if path == "." {
+		path = "/"
+	}
+	if path[:2] == "./" {
+		path = path[1:]
+	}
+	return filepath.Clean(path)
```

Comment:
should probably clean before you do those checks

Reply:
Switching the order breaks it so,,, I'm thinking I'll just leave it as is